### ISSUE TITLE
Fee reimbursement for trade 104007

### ISSUE THREAD

**Issue by Takoyakirorista** [bisq-network/support#1542]
![104007](https://github.com/bisq-network/support/assets/156135029/f3eff9e8-fc84-4c26-9af5-ece95708b1f2)

Maker fee transaction ID - 8c0609aa92feacd1c08c8ee76b1081a5f17563931b14a905d451492d48dfd9df

Taker fee transaction ID - d7c35ab37bccd6dbb265e2d04b38e7f0bdb17803899e04988e0ac89aaf285031

Deposit transaction ID - 56bebc5209d831098f11518162930c50c1de20a028ffa4dfbd5dd4ec10b6ae3b

Delayed payout transaction ID - d2e7a7ebb801af6958645014e6d846f7f678cb8103411e8061099c20c1451e6e

Bisq version: 1.9.14

BTC address for reimbursement: bc1quad3kj3seyga968x8eztpxjukkkgryuq4xsk96

**Comment by darawhelan** [comment]
Hi @Takoyakirorista 

This trade did not fail. It ended up in arbitration on 22 January 2024.

I would recommend moving the trade to 'Open Trades' and pressing 'Ctrl' and 'O' to open arbitration to speak to the arbitrator.

@refund-agent2 

If you are unable to contact them that way please reach out to support on Bisq 

https://bisq.wiki/Matrix_bisq.chat

**Comment by darawhelan** [comment]
I would also suggest you do an SPV resync

https://bisq.wiki/Resyncing_SPV_file

**Comment by refund-agent2** [comment]
I reimbursed the BTC buyer the trade amount and the security deposit. Buyer opened arbitration and replied, while BTC seller only said that the "payment is ok", but stopped replying to why is this trade in mediation, or if he accepted mediation suggestion.

Arbitration was opened the 17th jan and I replied the 18th, but as mining fees weren't enough, the Delayed Payout Transaction only confirmed on the 22nd jan. I paid 2 days after the DPT was confirmed and 7 days after buyer initiated arbitration.

There was no mediation suggestion and I did not find necessary to ask the mediator about details.
  
---
Ticket closed on Jan 24, 2024 7:57:54 AM
Arbitrator node address: 3z5jnirlccgxzoxc6zwkcgwj66bugvqplzf6z2iyd5oxifiaorhnanqd.onion:9999

Summary:
Trade ID: 104007
Currency: BRL
Reason for dispute: Peer was late
Payout suggestion: BTC buyer gets trade amount payout
Trade amount: 0.0080 BTC
Payout amount for BTC buyer: 0.00922160 BTC
Payout amount for BTC seller: 0.00 BTC

Summary notes:
The seller did not release the BTC, only acknowleged that the “payment is ok” but why buyer had to open arbitration is still unknown.


-----BEGIN SIGNATURE-----
302c021454f2d0efa4a49edb83df48c17903853bd84318200214624831eff4c894386562e2b9ae6599d59b6c17ab
-----END SIGNATURE-----
